app/controllers/conference_registrations_controller.rb

```diff
@@ -20,6 +20,11 @@ def new
       redirect_to edit_conference_conference_registrations_path(@conference.short_title)
     end
 
+    if @conference.registration_limit_exceeded?
+      redirect_to root_path, alert: 'Registration limit exceed'
```

Comment:
Shall we make the message a bit more specific?
"Sorry, registration limit exceeded for #{@conference.title}."